src/state/bridge/hooks.ts

```diff
@@ -17,13 +17,29 @@ import {
   isMultiErc20ToErc677BridgeToken
 } from '../../utils'
 import { formatEther, formatUnits } from 'ethers/lib/utils'
+import { BINANCE_CHAIN_ID } from '../../connectors'
+
+export enum BridgeType {
+  ETH_FUSE_NATIVE = 'ETH_FUSE_NATIVE',
+  ETH_FUSE_ERC677_TO_ERC677 = 'ETH_FUSE_ERC677_TO_ERC677',
+  ETH_FUSE_ERC20_TO_ERC677 = 'ETH_FUSE_ERC20_TO_ERC677',
```

Comment:
any difference in `ETH_FUSE_ERC20_TO_ERC677` and `BNB_FUSE_ERC20_TO_ERC677` except the contracts?
I think this 2 btidges can be AMB, nut with `BridgeDirection` we can conclude the network and contracts

Reply:
I like the idea, let me work on that